applications/crossbar/src/crossbar_init.erl

```diff
@@ -212,6 +216,36 @@ maybe_start_plaintext(Dispatch) ->
             end
     end.
 
+-spec get_binding_ip() -> inet:ip_address().
+get_binding_ip() ->
+    Default = case is_ipv6_supported() of
+                  'true' -> <<"::">>;
+                  'false' -> <<"0.0.0.0">>
+              end,
+    %% expilicty convert to list to allow save the default value in human readable value
+    IP = kz_util:to_list(kapps_config:get_binary(?CONFIG_CAT, <<"ip">>, Default)),
```

Comment:
@icehess if you create a `get_default_binding` fun and use it here instead of `Default`, then `ip` will not be marked as required